src/jats/importer/jats-body-dom-parser.ts

```diff
@@ -343,6 +343,24 @@ const nodes: NodeRule[] = [
       }
     },
   },
+  {
+    tag: 'p',
+    node: 'general_table_footnote',
+    context: 'footnotes_element/|table_element_footer/',
```

Comment:
What does that mean? Are you using this 'general_table_footnote' node to represent any **p** in the `table_element_footer` or `footnotes_element`? 

I was thinking that we would only have one 'general_table_footnote' per 'table_element_footer', and it would contain all the paragraphs that do not belong to the 'footnotes_element'

Reply:
I believe this modification may impact the `<p>` elements within `footnote_element`, which typically represent references or table footnotes. However, the desired outcome is to exclusively alter the type of all `<p>` elements that are within `table_element_footer` but not  within `footnotes_element`.
so we may have to update the context to be something like : 

context: 'table_element_footer/!footnotes_element'